internal/error.go

```diff
@@ -117,3 +117,7 @@ func (e *wrappedError) Error() string {
 func (e *wrappedError) Inner() error {
 	return e.inner
 }
+
+func (e *wrappedError) Unwrap() error {
```

Comment:
Oddly, `wrappedError` was missing the `Unwrap()` method, so functions like `IsTimeout` were not working on wrapped `ErrDeadlineWouldBeExceeded` errors.

Reply:
Good catch. Does that suggest `IsTimeout` does not return true on any wrapped timeout errors?

Reply:
`IsTimeout` can return `true` on a wrapped timeout error as long as the wrapping error implements an `Unwrap` method. I'm not sure of a way to detect wrapped timeout errors with wrappers that don't implement `Unwrap`.